tests/upload/test_views.py

```diff
@@ -100,7 +100,7 @@ def test_document_upload_unknown_type(client):
 
     assert response.status_code == 400
     assert response.json == {
-        'error': "Unsupported document type 'application/octet-stream'. Supported types are: ['application/pdf', 'text/csv', 'text/plain']" # noqa
+        'error': "Unsupported document type 'application/octet-stream'. Supported types are: ['application/pdf', 'text/csv', 'text/plain', 'application/msword', 'application/vnd.openxmlformats-officedocument.wordprocessingml.document', 'image/jpeg', 'image/png', 'application/vnd.ms-excel', 'application/vnd.openxmlformats-officedocument.spreadsheetml.sheet', 'application/vnd.apple.numbers']" # noqa
```

Comment:
It would be nice to simply reuse `ALLOWED_MIME_TYPES` in the test so we don't have to update the same list..

Reply:
Thought about that but I guess it defeats the purpose of the test if we reuse the same constant. Leaving it as-is for now.

Reply:
Testing a constant is testing a language feature. The way I see it, from the test name, it is meant to test uploads of unknown file types.

Reply:
I'm okay to test constants but if I change the constant (by adding/removing a MIME type) and I don't need to update tests, I think this is a code smell.

Reply:
That is a fair point. It feels like we are testing 2 different things though (behavior on unsupported types + the supported types). Ideally, we should have dedicated test(s) then for these.